metrics-core/src/main/java/com/codahale/metrics/CsvReporter.java

```diff
@@ -124,6 +125,7 @@ public CsvReporter build(File directory) {
     private final File directory;
     private final Locale locale;
     private final Clock clock;
+    private final HashMap<File, PrintWriter> printWriterHashMap = new HashMap<File, PrintWriter>();
```

Comment:
I think its much better to use a ConcurrentHashMap here in order to make sure that its thread safe?